Formula/mongo-c-driver.rb

```diff
@@ -0,0 +1,75 @@
+class MongoCDriver < Formula
+  desc "C driver for MongoDB"
+  homepage "https://github.com/mongodb/mongo-c-driver"
+  url "https://github.com/mongodb/mongo-c-driver/releases/download/1.5.0/mongo-c-driver-1.5.0.tar.gz"
+  sha256 "b9b7514052fe7ec40786d8fc22247890c97d2b322aa38c851bba986654164bd6"
+
+  bottle do
+    cellar :any
+    sha256 "1e3697dc3b2725914b720cb2aff628d1abeccfd26fc74ebe24bf3be83a5b5ba3" => :sierra
+    sha256 "f53836a2f6179cc4e65f6ef78c5a71792f9bd319676e9e8ef06d8fa5b4fafc5e" => :el_capitan
+    sha256 "c9fce4411ff385d090dc9e20529a045fa98b91801edba2732015f674fa581763" => :yosemite
+  end
+
+  mongo_c_driver_version = version
+  resource "libbson" do
```

Comment:
I'm getting a "resource defined more than once" error when trying to install. Does this `resource` need to be inside a `stable do` block to avoid it?

```
$ brew install mongo-c-driver
Error: Resource "libbson" is defined more than once
Please report this bug:
  https://git.io/brew-troubleshooting
/usr/local/Homebrew/Library/Homebrew/software_spec.rb:101:in `resource'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/mongo-c-driver.rb:26:in `block in <class:MongoCDriver>'
/usr/local/Homebrew/Library/Homebrew/formula.rb:2091:in `instance_eval'
/usr/local/Homebrew/Library/Homebrew/formula.rb:2091:in `head'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/mongo-c-driver.rb:20:in `<class:MongoCDriver>'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/mongo-c-driver.rb:1:in `load_formula'
```